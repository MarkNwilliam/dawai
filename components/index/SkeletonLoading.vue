<template>
  <v-col class="mt-12" cols="12" sm="12">
    <v-skeleton-loader
      class="slight-br my-3"
      max-width="auto"
      type="list-item-avatar-three-line"
    ></v-skeleton-loader>
    <v-skeleton-loader
      class="slight-br my-3"
      max-width="auto"
      type="list-item-avatar-three-line"
    ></v-skeleton-loader>
    <v-skeleton-loader
      class="slight-br my-3"
      max-width="auto"
      type="list-item-avatar-three-line"
    ></v-skeleton-loader>
    <v-skeleton-loader
      class="slight-br"
      max-width="auto"
      type="image"
    ></v-skeleton-loader>
  </v-col>
</template>
