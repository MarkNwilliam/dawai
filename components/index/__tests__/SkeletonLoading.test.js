// Libraries
import Vuetify from 'vuetify'

// Component
import { mount, createLocalVue } from '@vue/test-utils'
import SkeletonLoading from '../SkeletonLoading'

// Utilities
const localVue = createLocalVue()

describe('SkeletonLoading.vue', () => {
  let vuetify

  beforeEach(() => {
    vuetify = new Vuetify()
  })

  it('should mount the skeleton loaders and compare snapshots', (done) => {
    const wrapper = mount(SkeletonLoading, {
      localVue,
      vuetify
    })

    // With jest we can create snapshot files of the HTML output
    expect(wrapper.html()).toMatchSnapshot()
    done()
  })
})
