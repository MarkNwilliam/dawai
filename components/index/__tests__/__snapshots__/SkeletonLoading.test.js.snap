// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SkeletonLoading.vue should mount the skeleton loaders and compare snapshots 1`] = `"<v-col cols=\\"12\\" sm=\\"12\\" class=\\"mt-12\\"><v-skeleton-loader max-width=\\"auto\\" type=\\"list-item-avatar-three-line\\" class=\\"slight-br my-3\\"></v-skeleton-loader> <v-skeleton-loader max-width=\\"auto\\" type=\\"list-item-avatar-three-line\\" class=\\"slight-br my-3\\"></v-skeleton-loader> <v-skeleton-loader max-width=\\"auto\\" type=\\"list-item-avatar-three-line\\" class=\\"slight-br my-3\\"></v-skeleton-loader> <v-skeleton-loader max-width=\\"auto\\" type=\\"image\\" class=\\"slight-br\\"></v-skeleton-loader></v-col>"`;
